src/services/fileStorage/model.js

```diff
@@ -47,6 +60,17 @@ const fileSchema = new Schema({
 	storageFileName: { type: String },
 	thumbnail: { type: String },
 	thumbnailRequestToken: { type: String, default: uuid },
+	securityCheck: {
```

Comment:
the model allows to show only the latest check, do we need history here? @janrenz